config/config.go

```diff
@@ -827,8 +829,10 @@ func (cfg *StateSyncConfig) TrustHashBytes() []byte {
 // DefaultStateSyncConfig returns a default configuration for the state sync service
 func DefaultStateSyncConfig() *StateSyncConfig {
 	return &StateSyncConfig{
-		TrustPeriod:   168 * time.Hour,
-		DiscoveryTime: 15 * time.Second,
+		TrustPeriod:         168 * time.Hour,
+		DiscoveryTime:       15 * time.Second,
+		ChunkRequestTimeout: time.Minute,
+		ChunkFetchers:       1,
```

Comment:
Are you sure we want to take these defaults. Having 1 chunk fetcher means that the process is no longer done in parallel which could really slow down state sync.

Reply:
Yeah, good catch. We should keep the original values (10s/4).